### PR TITLE
Edit .gitignore and revise hi-res textures section in mupen64plus.md

### PR DESCRIPTION
**Pull request by kynoptic**
**Changes introduced**:
- `.gitignore` has been updated to ignore `.DS_Store` files across the repository.
- The high-resolution texture section in `mupen64plus.md` has been reworked to improve readability and split the guide into two parts:
     - Pre-compiled textures
     - Compiling textures oneself

### CODE REVIEW

**Comment by kynoptic** [comment]
I would love to better understand which core options must match those set when the textures were compiled so I can include that in the documentation. Any help figuring that out is much appreciated.

File: docs/library/mupen64plus.md [hunk 1]
```diff
@@ -104,23 +104,40 @@ The Mupen64plus-Next core saves/loads to/from these directories.
 - The Mupen64plus-Next core's internal sample rate is 44100 Hz
 - The Mupen64plus-Next core's core provided aspect ratio is (Ratio)
 
-###  Hires Textures
+### High-resolution textures
```
Comment (RobLoach) [inline docs/library/mupen64plus.md]:
``` suggestion
### High-Resolution Textures
```

File: docs/library/mupen64plus.md [hunk 2]
```diff
@@ -104,23 +104,40 @@ The Mupen64plus-Next core saves/loads to/from these directories.
...
+2. By compiling your own high-resolution texture packs from uncompressed Rice sources.
 
-Recommended Core Options for pre-compiled very popular .htc texture packs, such as those of Djipi or Mollymutt, are:
-- mupen64plus-EnableTextureCache = "True"
-- mupen64plus-txHiresEnable = "True"
-- mupen64plus-txCacheCompression = "True"
-- mupen64plus-txHiresFullAlphaChannel = "False"
-- mupen64plus-EnableEnhancedTextureStorage = "False"
-- mupen64plus-EnableEnhancedHighResStorage = "False"
+#### Use pre-compiled high-resolution textures
```
Comment (RobLoach) [inline docs/library/mupen64plus.md]:
``` suggestion
#### Use Pre-Compiled High-Resolution Textures
```

File: docs/library/mupen64plus.md [hunk 3]
```diff
@@ -104,23 +104,40 @@ The Mupen64plus-Next core saves/loads to/from these directories.
...
+**Recommended core options** for widely used `.htc` texture packs (for example , those used by Djipi or Mollymutt):
+
+- `mupen64plus-EnableTextureCache = "True"`
+- `mupen64plus-txHiresEnable = "True"`
+- `mupen64plus-txCacheCompression = "True"`
+- `mupen64plus-txHiresFullAlphaChannel = "False"`
+- `mupen64plus-EnableEnhancedTextureStorage = "False"`
+- `mupen64plus-EnableEnhancedHighResStorage = "False"`
+
+#### Compile your own high-resolution texture packs
```
Comment (RobLoach) [inline docs/library/mupen64plus.md]:
``` suggestion
#### Compile Your Own High-Resolution Texture Packs
```

**Review by RobLoach — APPROVED** [review]
Makes sense to me!

File: docs/library/mupen64plus.md [hunk 4]
```diff
@@ -104,23 +104,40 @@ The Mupen64plus-Next core saves/loads to/from these directories.
...
-If you compile your own texture pack, you should be sure to enable the further three for Alpha Channel and Enhanced Storage, which will then produce a hires texture pack with the *.hts extension.
-Compilation under Windows seems to be more complicated than under Linux or iOS. Rice texture packs must be converted to 32bit PNG.
+- Place the `.htc` formatted textures into the `Mupen64plus/cache` directory.
+- Ensure the `.htc` file name corresponds to the system name of the game as shown in the mupen64plus console.
+- Pre-compiled high-resolution packs will work only if Retroarch core settings related to textures match those used during the pack's compilation. To avoid potential mismatches, you might prefer to compile your own texture packs.
+
+**Recommended core options** for widely used `.htc` texture packs (for example , those used by Djipi or Mollymutt):
```
Comment (nighto) [inline docs/library/mupen64plus.md]: teeny-tiny suggestion: remove space before comma

**Pull request merged**